### ISSUE TITLE
Does  it  support TableWidget or GridWidget?

### ISSUE THREAD

**Issue by coolsnake** [flexxui/flexx#538]
Table Widget or Grid Widget can show data in web site application.
They can dynamically increase  rows.

I also want some thing  like Navbar Widget. There are some entries in the nav bar. If We click the one entry, the corresponding content will display in TableWidget or GridWidget in the right area.

**Comment by almarklein** [comment]
No, Flexx does not (yet) have widgets for laying things out in a grid. There is the `FormLayout` which covers some use-cases. A widget to display data in a table should not be too hard to make using `<table>`.

**Comment by coolsnake** [comment]
Thanks. For the NavBar, I looked into TabLayout and tried  to align the title in one column on left side, but failed.

**Comment by almarklein** [comment]
#553 implements a GridWidget

**Comment by damienflament** [comment]
I'm implementing a `TableWidget` for my own usage, while making it generic.
I'm trying to provide an API as elegant as other widgets provided by Flexx with support for optional header and optional caption.

I will post a PR when done.

**Comment by almarklein** [comment]
That sounds good! I am planning for a release this week. With a bit of luck we can get that in as well :)

**Comment by h5rdly** [comment]
This is awesome. 

@damienflament did you encounter bokeh's [DataTable](https://github.com/bokeh/bokeh/blob/master/bokeh/models/widgets/tables.py)? It uses a ColumnDataSource for a "backend".

I believe the mechanics might be somewhat similar to those of flexx - bokeh uses pscript as well.

**Comment by Kazakoff64** [comment]
Hey, any traffic?

**Comment by almarklein** [comment]
There could be. There has been some work in #554, but that implementation uses subwidgets for the cells. A widget capable of dealing with a lot of rows should not use sub-widgets, but directly use `<table>`. So far no-one has tried to implement that yet.
